carbon-design-system/carbon
# Feat(codesnippet): add experimental component Code Snippet, Closes #1350

Closes #1350 

#### Changelog

**New**
- Experimental Code Snippet
- Add styling for copy feedback (was not using Copy Button styling in experimental component)

**Removed**
- Line 44 in `code-snippet.js` that added expand class on init (multi line code would init expanded)

#### Testing / Reviewing
- Ensure multi line code snippet is _not_ expanded on init in both Experimental/Standard
- Ensure copy feedback is the correct color in both Experimental/Standard
- Staging link: http://carbon-dev-environment-cs-xx-noisy-meerkat.mybluemix.net/?nav=code-snippet
